### PR TITLE
Added Automotive to our nav

### PR DESCRIPTION
**Pull request by weyermannx**
🐘 🎨 
screenshot 2016-08-10 11 44 35

### CODE REVIEW

**Comment by weyermannx** [comment]
:wave:

**Comment by rubene** [comment]
:book:

**Comment by rubene** [comment]
💩  increate gem version

**Comment by weyermannx** [comment]
:wave:

**Comment by rubene** [comment]
🐘 🐘 🐘

**Comment by darkodosenovic** [comment]
👋

**Comment by kaytcampbell** [comment]
:art: :art: :art:

File: lib/ama_layout/navigation.yml [hunk 1]
```diff
@@ -14,6 +14,12 @@ member:
         link: "<%= Rails.configuration.youraccount_site %>/billing"
       - text: "Print/Request New Card"
         link: "<%= Rails.configuration.youraccount_site %>/membership"
+  - text: "My Roadside Assistance"
+    link: "<%= Rails.configuration.automotive_site %>/"
```
Comment (mvandenbeuken) [inline lib/ama_layout/navigation.yml]: I guess we'll have to add this to application.rb for each app with the nav, and to the dev, staging and prod envs for each app as well.

File: spec/ama_layout/navigation_spec.rb [hunk 2]
```diff
@@ -62,8 +64,17 @@
 
     context "member" do
       context "subnavs" do
+        context "registries" do
```
Comment (mvandenbeuken) [inline spec/ama_layout/navigation_spec.rb]: automotive

**Comment by mvandenbeuken** [comment]
🐘 🐘 🐘

**Comment by darkodosenovic** [comment]
:wave:

**Comment by mvandenbeuken** [comment]
🐘 🐘 🐘

**Comment by darkodosenovic** [comment]
:wave:

**Comment by jessedoyle** [comment]
:elephant::elephant::elephant:

**Pull request merged**